### PR TITLE
Update docs

### LINKED ISSUE #16112: [OpenForce 2022] Optimize official documents
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### Aim

Check the source file below, search `SQL`  word in `SQLExecutionHook ` and  keep a space between Chinese and English words.

Source file:
https://github.com/apache/shardingsphere/blob/master/docs/document/content/dev-manual/kernel.cn.md

Official document link:
https://shardingsphere.apache.org/document/current/cn/dev-manual/kernel/

### PR DESCRIPTION
**Pull request by RakeshPotnuru**
Fixes #16112

Changes proposed in this pull request:
- Optimize docs by adding space between English and Chinese words

### CODE REVIEW

**Comment by menghaoranss** [comment]
@RakeshPotnuru thanks for your contribution.

File: docs/document/content/dev-manual/kernel.cn.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ chapter = true
...
-| SQLExecutionHook              | SQL执行过程监听器         |
+| *SPI 名称*                     | *详细说明*      |
+| ----------------------------- |-------------|
+| SQLExecutionHook              | SQL 执行过程监听器 |
 
-| *已知实现类*                    | *详细说明*               |
-| ----------------------------- | ----------------------- |
-| TransactionalSQLExecutionHook | 基于事务的SQL执行过程监听器 |
+| *已知实现类*                    | *详细说明*            |
+| ----------------------------- |-------------------|
```
Comment (yx9o) [inline docs/document/content/dev-manual/kernel.cn.md:40]: The spaces between the symbols `|-` are consistent with other codes.

Comment (RakeshPotnuru) [reply docs/document/content/dev-manual/kernel.cn.md:40]: @yx9o 
I didn't get you

Comment (yx9o) [reply docs/document/content/dev-manual/kernel.cn.md:40]: Space between `|` and `-`.

Comment (yx9o) [reply docs/document/content/dev-manual/kernel.cn.md:40]: image

**Comment by yx9o** [comment]
@RakeshPotnuru great job.

**Review by yx9o — APPROVED** [review]

**Pull request merged**